curator-recipes/src/main/java/org/apache/curator/framework/recipes/nodes/GroupData.java

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Can't ChildData be re-used here? The only difference is the stat object which doesn't hurt for GroupMember.